### DISCUSSION TITLE
Pi4 Unbuntu Arch64, tini exec format error?

### DISCUSSION THREAD

**Discussion by ciordia9** [activexray/Doplarr#123] [Q&A]
```
Starting doplarr ... done
Attaching to doplarr
doplarr             | exec /sbin/tini: exec format error
doplarr exited with code 1
```

Thoughts?

**Comment by activexray** [comment]
Is this with one of my docker containers? I don't think I built for aarch64. You could just run the jar directly

**Reply by ciordia9** [reply]
Yes, this is using the doplarr yml setup in your instructions. I try and keep most of my homelab stuff on my pi4 in containers. I'll circle back and try for the jar.
